src/Context/WebApiContext.php

```diff
@@ -59,6 +59,14 @@ public function setClient(Client $client)
     }
 
     /**
+     * @return \GuzzleHttp\Message\ResponseInterface
+     */
+    public function getResponse()
```

Comment:
Maybe you could do the same of #27 here:

```php
protected function getResponse()
{
    if (! $this->response) {
        throw new Exception("You must first make a request to check a response.");
    }

    return $this->response;
}
```

I don't understand why this was not even reviewed, it has 23 comments and almost a year here.
This feature is very useful for custom contexts... Any way, I will probably copy WebApiContext and use it as my base context until this merge... :cry: